cloud/aws/bin/dumpdb.py

```diff
@@ -0,0 +1,180 @@
+import os
+import ipaddress
+import sys
+import tempfile
+import shlex
+import subprocess
+from time import sleep
+from datetime import datetime
+import urllib.request
+
+from cloud.aws.templates.aws_oidc.bin import resources
+from cloud.aws.templates.aws_oidc.bin.aws_cli import AwsCli
+from cloud.shared.bin.lib import terraform
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+from cloud.shared.bin.lib.print import print
+
+
+def run(config: ConfigLoader):
+    aws = AwsCli(config)
+    print(
+        "\n"
+        "\033[31m!!! WARNING !!!: This command will create a dump of the entire database, including personally identifiable information (PII). Ensure you take the utmost care in handling this data and store it in a secure location.\033[0m\n"
```

Comment:
Nit: the usual recommendation of output a block of text is to use triple-quotes and `textwrap.dedent`:

```
print(textwrap.dedent("""
    Note this is a long
    multi-line
    message
"""))
```